doc/matter/conf.py

```diff
@@ -42,12 +42,12 @@
 html_last_updated_fmt = "%b %d, %Y"
 html_show_sourcelink = True
 html_show_sphinx = False
-html_title = "Matter documentation (nRF Connect SDK)"
+html_title = "Matter SDK documentation (nRF Connect platform)"
 
 html_theme_options = {
     "docset": "matter",
     "docsets": utils.ALL_DOCSETS,
-    "subtitle": "nRF Connect SDK",
+    "subtitle": "nRF Connect platform",
```

Comment:
FYI: This refers to the Matter implementation platforms: https://project-chip.github.io/connectedhomeip-doc/guides/index.html

Reply:
Where specifically is this tag used used? I don't see any mention of "nRF Connect SDK" on the linked page, and I'm not sure the linked page is part of our build of the Matter docs.

Reply:
No no, the other way around: `nRF Connect platform` refers to the nRF Connect platform on https://project-chip.github.io/connectedhomeip-doc/guides/index.html

`nRF Connect SDK` is this string, which I want changed to `... platform`:
![image](https://github.com/nrfconnect/sdk-nrf/assets/41291385/fe275b46-e26e-4779-9e47-6f9e02d485d8)